src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messages.html.twig

```diff
@@ -31,7 +31,13 @@
         <tbody>
         {% for message in collector.messages %}
         <tr>
-            <td>{{ message.message.type }}</td>
+            <td>
+                {{ message.message.type }}
```

Comment:
I don't believe we still need to display the type anymore if we use the dumper, as the FQCN is available as overlay :)